bundles/org.openhab.binding.somneo/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -96,7 +165,7 @@
 			<channel id="breathingRate" typeId="breathingRate"/>
 			<channel id="duration" typeId="relaxDuration"/>
 			<channel id="guidanceType" typeId="guidanceType"/>
-			<channel id="lightIntensity" typeId="system.brightness">
+			<channel id="lightIntensity" typeId="sunriseBrightness">
```

Comment:
Would you like to use the label and description of the channel type now or still overwrite it here?

Also, since the channel type changed, you need to provide update instructions for the channel. See https://next.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

Update instructions also apply to new channels added to existing things.